app/helpers/application_helper/toolbar/physical_storages_center.rb

```diff
@@ -19,6 +19,12 @@ class ApplicationHelper::Toolbar::PhysicalStoragesCenter < ApplicationHelper::To
             :confirm => N_("Refresh relationships and power states for all items related to these Physical Storages?"),
             :options => {:feature => :refresh}
           ),
+          button(
+            :physical_storage_new,
+            'pficon pficon-add-circle-o fa-lg',
+            t = N_('Attach a new storage system'),
```

Comment:
I suggest to add `:url   => "/new",` here, then we will not need https://github.com/ManageIQ/manageiq-ui-classic/pull/7407/files#diff-d0898d90cc765a6035529968632ba627R14-R22 , https://github.com/ManageIQ/manageiq-ui-classic/pull/7407/files#diff-d0898d90cc765a6035529968632ba627R7 , https://github.com/ManageIQ/manageiq-ui-classic/pull/7407/files#diff-c32dcaa7dedd2b85b7ee0ce6036b3f54R267
@skateman what do you think, that's the approach we are taking with de-explorization.

Reply:
Definitely, the simpler the better!